client/analysis/analysis.go

```diff
@@ -27,8 +27,9 @@ func StartAnalysis() (string, error) {
 	requestPayload := types.JSONPayload{
 		RepositoryURL:    config.RepositoryURL,
 		RepositoryBranch: config.RepositoryBranch,
+		LanguageExclusions: config.LanguageExclusions,
 	}
-
+	
```

Comment:
Pro tip if coding in Visual Studio: https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces

Reply:
Nice tip! Trailing spaces should be removed in last commit :D